astropy/io/fits/_tiled_compression/codecs.py

```diff
@@ -34,9 +34,64 @@ class Codec:
     "Rice1",
     "PLIO1",
     "HCompress1",
+    "NoCompress",
 ]
 
 
+def _as_big_endian_array(data):
+    return data.astype(np.asarray(data).dtype.newbyteorder(">"))
```

Comment:
@astrofrog isn't this going to copy the array?

Reply:
Good catch need copy=False

Reply:
Fixed (was done in the NOCOMPRESS PR)